src/ctia/lib/es/index.clj

```diff
@@ -9,12 +11,19 @@
    [ctia.events.producers.es.mapping :refer [producer-mappings]]
```

Comment:
I think ideally a lib namespace should not use other CTIA namespaces like this.  A lib should be low-level and should not depend on higher level things.  Maybe this isn't a lib, or maybe parts of it are not.  Maybe this lib should just take parameters instead of being coupled.